cocos/audio/AudioEngine.cpp

```diff
@@ -152,6 +154,9 @@ class AudioEngine::AudioEngineThreadPool
 
 void AudioEngine::end()
 {
+#if (CC_TARGET_PLATFORM == CC_PLATFORM_OHOS)
+    stopAll();
```

Comment:
Why does OHOS need to invoke `stopAll` in `end` but other platforms don't?

Reply:
recovered